io/disk/Avago_storage_adapter/avago9361_vd.py

```diff
@@ -99,16 +108,72 @@ def test_maxvd(self):
                     self.vd_details()
                 self.vd_delete()
 
+    def test_init(self):
+
+        """
+        Test case to start Fast and Full init on VD
+        """
+        self.vd_create('WB', 'ra', 'cached', 256)
```

Comment:
I have wantedly not written vd_details under vd_create. Because, in future its better, if we have to see only details it will be easy.